src/UseCases/Embed/EmbedRequestBuilder.php

```diff
@@ -15,12 +17,17 @@ public static function argumentsToRequest( array $arguments, bool $showEditButto
 		$language = $normalizedArguments['lang'] ?? null;
 		$line = $normalizedArguments['line'] ?? null;
 		$specificLines = $normalizedArguments['specificLines'] ?? null;
+		$render = $normalizedArguments['render'] ?? null;
+
+		/** @var bool */
```

Comment:
Generally we don't typehint variables like this, unless it is something like an array where you need to provide additional info.

In this case you can do an explicit comparison:
```php
MediaWikiServices::getInstance()->getMainConfig()->get( 'ExternalContentRenderMarkdownByDefault' ) === true;
```
Because that forces the result to be boolean. (This is not the same as comparing a boolean to true like in a previous PR.)

Otherwise, if the user sets invalid config in LocalSettings:
```php
$wgExternalContentRenderMarkdownByDefault = 'abc';
```
Then the variable isn't actually a boolean and it breaks:
![Screenshot_20231027_135038](https://github.com/ProfessionalWiki/ExternalContent/assets/1428594/dda400e3-f883-4f58-a995-c01cbf04efb2)